guide/english/wordpress/index.md

```diff
@@ -23,19 +23,19 @@ Just to name a few advantages of WordPress:
 * Has mobile app (Android and iOS), which can be used to post and manage their Wordpress webpage.
 * Wordpress sites can be hosted on any server running the LAMP stack
 * WordPress themes make it possible for non-designers to create a customized website
-* Wordpress site are very scalable, you can start as simple blog then end up with complex online stores, membership, and affiliate program
+* Wordpress site is very scalable, you can start an as simple blog then end up with complex online stores, membership, and affiliate program
 
-As a CMS, WordPress allows you to control and manage content of your website with a very easy system, such as the WordPress dashboard panel. A dashboard lets you to do work on your website without needing to program anything. You can add or delete images and edit text on your webpage fast and easily.
+As a CMS, WordPress allows you to control and manage the content of your website with a very easy system, such as the WordPress dashboard panel. A dashboard lets you do work on your website without needing to program anything. You can add or delete images and edit text on your webpage fast and easily.
 
 Whether it is page transitions or a customized contact form, WordPress users are only a few clicks away from success and a beautiful website.
 
 ### WordPress Community
 
 If you would like to meet new people who are interested in Wordpress there are many meetups around the world including huge events called WordCamp which can be a great place to network. 
 
-### WordPress as Blogging Platform
+### WordPress as a Blogging Platform
 
-Blogging is a foundational feature of WordPress. One of the primary things it allows you to do is create individual "posts", and designate a main blog page that displays them in chronological order.
+Blogging is a foundational feature of WordPress. One of the primary things it allows you to do is create individual "posts", and designate the main blog page that displays them in chronological order.
```

Comment:
`a` is fine here because the `main blog page` is not a proper noun as we are referring to a main page of any blog there is, so please revert this back.